ec2/spark_ec2.py

```diff
@@ -70,7 +70,7 @@
     "1.2.1",
     "1.3.0",
     "1.3.1",
-    "1.4.0"
+    "1.4.0",
```

Comment:
I also do this when the language syntax allows, since it means that adding a next line will not show up as a change (because of adding the comma to the previous line) but just a 1-line addition. Really minor but has some tiny value.